climada/hazard/centroids/centr.py

```diff
@@ -574,16 +575,29 @@ def set_area_approx(self, min_resol=1.0e-8):
             LOGGER.error('Pixel area of points can not be computed.')
             raise ValueError
 
-    def set_dist_coast(self, signed=False, precomputed=False, scheduler=None):
-        """Set dist_coast attribute for every pixel or point. Distance to
-        coast is computed in meters.
+    def set_elevation(self, topo_path):
```

Comment:
This function might come in handy in the future. It sets the `elevation` property from a given elevation raster file. Currently, it's only used in the tutorial notebook for illustration purposes.